### PR TITLE
add inital code to start work on playback

### PR DESCRIPTION
**Pull request by birdage**
@ocefpaf this is beginning of something great. lets leave it open

### CODE REVIEW

**Comment by ocefpaf** [comment]
> @ocefpaf this is beginning of something great. 

@birdage indeed!  I will review and we can keep working in it.

File: folium/folium.py [hunk 1]
```diff
@@ -58,7 +63,7 @@ class Map(object):
     def __init__(self, location=None, width=960, height=500,
                  tiles='OpenStreetMap', API_key=None, max_zoom=18, min_zoom=1,
                  zoom_start=10, attr=None, min_lat=-90, max_lat=90,
-                 min_lon=-180, max_lon=180):
+                 min_lon=-180, max_lon=180,playback=False):
```
Comment (ocefpaf) [inline folium/folium.py]: I am starting to get a little bit worried about the kwargs in the `__init__`.  We'll end up like `pandas.read_wathever` tons of options :smile:

Maybe we can brainstorm a better way to start maps, something that we could pass the advanced options/objects and added them to the simple map.

File: folium/templates/fol_template.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.css" />
    <script src="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"></script>
 
-   <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```
Comment (ocefpaf) [inline folium/templates/fol_template.html]: remove the `https`

File: folium/templates/fol_template.html [hunk 3]
```diff
@@ -21,6 +21,22 @@
...
+
+    <link href="http://code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css" rel="stylesheet" type="text/css" />    
+
+    <link href="http://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />    
+    
+    <link href="http://jdewit.github.io/bootstrap-timepicker/css/bootstrap-timepicker.min.css" rel="stylesheet" type="text/css" />
+
+   <script src="https://raw.githubusercontent.com/hallahan/LeafletPlayback/master/examples/example_2_control.js"></script>
+
+   <link rel="stylesheet" href="https://raw.githubusercontent.com/hallahan/LeafletPlayback/master/examples/example_2.css">
```
Comment (ocefpaf) [inline folium/templates/fol_template.html]: All these `http(s)` shoudl go away.

**Comment by ocefpaf** [comment]
PS: Lets Ignore the travis failure until we have a final template.

File: folium/templates/fol_template.html (hunk 2 above)
Comment (birdage) [inline folium/templates/fol_template.html]: THANKS!

File: folium/templates/fol_template.html (hunk 3 above)
Comment (birdage) [inline folium/templates/fol_template.html]: yea yea

**Comment by birdage** [comment]
@ocefpaf this is just to start getting it on the map, it will need alot of refining and might take a while to get finished, just trying to get it to work for now :smile:

**Comment by ocefpaf** [comment]
Take your time.  Even if we define a new logic for adding features like this in the future lets can get this working using the current approach (adding `kw` to `__init__` and URLs to `fol_template.html`).

**Comment by themiurgo** [comment]
This would be a great addition, waiting impatiently for this. :satisfied:

**Comment by ocefpaf** [comment]
@themiurgo we are discussing about implementing #125 instead.  What do you think?

**Pull request closed without merging**